setup.py

```diff
@@ -9,6 +10,21 @@
     changes = f.read()
 
 
+install_requires = [
+    'pytest',
+    'pep8',
+    'pyflakes',
+    'coverage',
+    'sphinx',
+    'pytest-cov',
+]
+
+if version_info.major == 2:
+    install_requires.append([
+        'mock',
+        'future',
+    ])
```

Comment:
I couldn't find one, but if so great.
I did find an example of how to describe "test_requires" in setup.py instead of install_requires and launch pytest from `setup.py test` (which actually triggers fetching the packages if they aren't already available):
https://github.com/jaraco/jaraco.collections/blob/master/setup.py
But when I've tried that out it seems to run more slowly, and consistently fails on first run on Python 2.7, but then works fine. (packages do get installed into ./.eggs instead of into a site-packages or user dir, I think that might be a benefit?)